test/inductor/test_torchinductor.py

```diff
@@ -7151,6 +7153,40 @@ def foo(m, inp):
             self.assertTrue(
                 foo(m, inp)[0].is_contiguous(memory_format=torch.channels_last)
             )
+        
+        # So far these 2 tests just assert that nothing broke
+        # But ideally I'd like a a test that checks for whether a string is present in the IR
+        # I wonder if there's a better way to do this
+        @requires_cuda()
```

Comment:
@Chillee any feedback on what's a good test setup - in my notebook what i was doing to validate things was
1. `gm.print_readable()` and making sure new padding op and slicing ops are there
2. `gm.forward()` still works with the same input shape
3. I could also quickly validate that it makes things faster

And I figure to test 
1. I can either test that the compiled model  works with shape padding so more e2e - i couldnt find any existing tests for the existign shape padding implementation
2. And I can test the pass in a standalone way using functorch

The fx/passes directory didn't have enough passes for me to justify putting things there

Reply:
Yeah that's mostly fine.